### ISSUE TITLE
Problem when adding Buefy's CSS file before other CSS files

### ISSUE THREAD

**Issue by lobo-tuerto** [buefy/buefy#591]
### Overview of the problem

A brand new Vue.js project that includes Buefy and MDI (Material Design Icons) will fail to build if Buefy's CSS file is included before MDI's CSS file.

I'm new to Vue.js, so excuse me if I'm doing something silly. I created a new app, then added `buefy` and `mdi` to my project. then included their CSS files like this on `/index.js`:

```js
import 'buefy/lib/buefy.css'
import 'mdi/css/materialdesignicons.css'
```

**Buefy** version: [0.6.3]
**Vuejs** version: [2.5.13]
**OS/Browser**: Linux 4.14.16-1-MANJARO x86_64 GNU/Linux

### Description

When trying to build, the output is this:

```bash
$ yarn build
yarn run v1.3.2
$ node build/build.js
⠸ building for production...Error processing file: static/css/app.db19096df849c1b4cc99d3b8bab7a1ca.css
(node:5053) UnhandledPromiseRejectionWarning: SyntaxError: Unexpected token � in JSON at position 111003
    at JSON.parse (<anonymous>)
    at new SourceMapConsumer
```

### Steps to reproduce

I setup a repo showcasing the problem here: https://github.com/lobo-tuerto/buefy-css-build-problem

1. Clone the repo
2. `yarn install`
3. `yarn build`

### Expected behavior

It should build the app normally.

### Actual behavior

It fails to build because `buefy/lib/buefy.css` and `buefy/lib/buefy.min.css` are missing a newline character at the end-of-file.

### Workarounds

I found several workarounds to this problem:

* Instead of:

  ```bash
  import 'buefy/lib/buefy.css'
  import 'mdi/css/materialdesignicons.css'
  ```

  Do:

  ```bash
  import 'mdi/css/materialdesignicons.css'
  import 'buefy/lib/buefy.css'
  ```

  Basically this workaround consists on including Buefy's CSS at the end so it doesn't mess up with other libraries' CSS files that include comments at the start.

* Remove CSS comment at the top of `mdi/css/materialdesignicons.css`.
  If you remove the first line (a comment), it'll build OK.
* Add a blank line at the end of `buefy/lib/buefy.css` and `buefy/lib/buefy.min.css`.
  This looks like the right thing to do.
* Disable production source maps in `/config/index.js` (`build.productionSourceMap: false`).
  Hmmm.

**Comment by gallowdp** [comment]
I also have this problem and am unable to solve by swapping the import order. I still get the same error.

**Comment by lobo-tuerto** [comment]
@gallowdp It has to be the last line when importing CSSs.

Or try one of the other 3 workarounds in the meantime while we get an official response from the maintainer.

**Comment by JeremyGranger** [comment]
Just ran into this issue. @lobo-tuerto your first workaround (swapping the css imports) worked for me. Thanks so much! What an annoying error...
